### DISCUSSION TITLE
type Address has been removed from wagmi v2?

### DISCUSSION THREAD

**Discussion by linghuccc** [wevm/wagmi#3464] [Q&A]
Migrate from v1 to v2 (https://wagmi.sh/react/guides/migrate-from-v1-to-v2) did not mention anything about "type Address". But when I tried to "import { Address } from 'wagmi'", below error is shown:
Module '"wagmi"' has no exported member 'Address'.

I tried to change 'Address' to 'address' and it still did not work. Please advise. Thank you.

**Comment by tmm** [comment]
You can import from Viem. Feel free to submit a PR updating the guide as well.

```ts
import { type Address } from 'viem'
```
